### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0xMuluh**
improve badges. Any suggestions for other badges? perhaps slack chat? years in bioc?
@antagomir

### CODE REVIEW

**Comment by antagomir** [comment]
Slack maybe not but if you see useful Bioc badges we could add some of those.

File: README.md [hunk 1]
```diff
@@ -2,10 +2,13 @@
 
 <!-- badges: start -->
 
-[![R-CMD-check-bioc-devel](https://github.com/microbiome/mia/workflows/R-CMD-check-bioc-devel/badge.svg)](https://github.com/microbiome/mia/actions)
-<!--[![R-CMD-check-bioc](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml/badge.svg)](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml)-->
+[![Platforms](http://bioconductor.org/shields/availability/release/miaViz.svg)](https://bioconductor.org/packages/release/bioc/html/miaViz.html#archives)
+[![rworkflows](https://github.com/microbiome/mia/workflows/rworkflows/badge.svg)](https://github.com/microbiome/mia/actions)
+[![Bioc-release](http://bioconductor.org/shields/build/release/bioc/mia.svg)](http://bioconductor.org/packages/release/bioc/html/mia.html)
```
Comment (TuomasBorman) [inline README.md:7]: You could also add Bioc devel

File: README.md [hunk 2]
```diff
@@ -2,10 +2,13 @@
 
 <!-- badges: start -->
 
-[![R-CMD-check-bioc-devel](https://github.com/microbiome/mia/workflows/R-CMD-check-bioc-devel/badge.svg)](https://github.com/microbiome/mia/actions)
-<!--[![R-CMD-check-bioc](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml/badge.svg)](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml)-->
+[![Platforms](http://bioconductor.org/shields/availability/release/miaViz.svg)](https://bioconductor.org/packages/release/bioc/html/miaViz.html#archives)
```
Comment (TuomasBorman) [inline README.md]: This badge is for miaViz

File: README.md [hunk 3]
```diff
@@ -2,10 +2,13 @@
...
+[![rworkflows](https://github.com/microbiome/mia/workflows/rworkflows/badge.svg)](https://github.com/microbiome/mia/actions)
+[![Bioc-release](http://bioconductor.org/shields/build/release/bioc/mia.svg)](http://bioconductor.org/packages/release/bioc/html/mia.html)
+[![Bioc-age](http://bioconductor.org/shields/years-in-bioc/mia.svg)](https://bioconductor.org/packages/release/bioc/html/mia.html#since)
 [![Codecov test
 coverage](https://codecov.io/gh/microbiome/mia/branch/master/graph/badge.svg)](https://codecov.io/gh/microbiome/mia?branch=master)
```
Comment (TuomasBorman) [inline README.md]: mia does not have master branch anymore. It is devel

File: README.md [hunk 4]
```diff
@@ -2,10 +2,13 @@
 
 <!-- badges: start -->
 
-[![R-CMD-check-bioc-devel](https://github.com/microbiome/mia/workflows/R-CMD-check-bioc-devel/badge.svg)](https://github.com/microbiome/mia/actions)
-<!--[![R-CMD-check-bioc](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml/badge.svg)](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml)-->
+[![Platforms](http://bioconductor.org/shields/availability/release/miaViz.svg)](https://bioconductor.org/packages/release/bioc/html/miaViz.html#archives)
+[![rworkflows](https://github.com/microbiome/mia/workflows/rworkflows/badge.svg)](https://github.com/microbiome/mia/actions)
```
Comment (TuomasBorman) [inline README.md]: I am wondering if this badge shows the most recent action ran in mia repo. If so, then this takes into account also those ran in other branches in devel.

For instance, if action fails in "branch_X", that is not informative for user. --> Can you check if we can only show the action in devel branch?

File: README.md (hunk 1 above)
Comment (TuomasBorman) [reply README.md:7]: Well, it seems that these badges have same same name, so user cannot distinguish Bioc devel / Bioc release.

Do not add devel

File: README.md (hunk 3 above)
Comment (TuomasBorman) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (TuomasBorman) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (0xMuluh) [reply README.md]: Ack.

**Pull request merged**